settings.gradle

```diff
@@ -10,3 +10,4 @@ rootProject.name = "appcues-android-sdk"
 
 include ':samples:kotlin-android-app'
 include ':appcues'
+include ':libs:dime'
```

Comment:
I'm not sure or not if having multi-module inside of our library/sdk project will add more complexity, maybe not. Is it more appropriate for an app to use than a library? I really like a lot of the concepts and simplifications here, just don't want to add any complexity to our project and deploy structures if we can avoid it - what is the downside of just including the `di` aspects in a package inside of the core sdk? I don't think there is really an outside re-use case for this appcues specific stuff.

Reply:
this will not add any complexity on the customers end, it may add a bit on. CI/TEST/JACOCO maybe depending on what job we are running (need to double check this) 

we could have it all in di package like we have most of our stuff today, I have a personal preference to start going on the opposite direction, better defining the running parts of our sdk as separate modules

https://developer.android.com/topic/modularization

Reply:
yeah 100% on board with modularization of an _app_ project. But I'm not sure on an SDK/lib project yet, maybe, just seems like it could add unnecessary complexity to something that is a redistributable package, if it was not truly a separable module with standalone value.